<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FansTab</class>
 <widget class="QWidget" name="FansTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>923</width>
    <height>492</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,1,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,0,0">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Armoury Crate profile:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="arCrateProfileComboBox">
       <property name="styleSheet">
        <string notr="true">width : 9em</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="defaultFanCurves">
       <property name="text">
        <string>Use default fan curves</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Fan curve:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="fanCurveComboBox">
       <property name="styleSheet">
        <string notr="true">width : 9em</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="cpuGroup">
     <property name="title">
      <string>CPU</string>
     </property>
     <layout class="QHBoxLayout" stretch="0,0,0,0,0,0,0,0,1">
      <item>
       <layout class="QVBoxLayout">
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="cpuTemp_0">
          <property name="text">
           <string>35°C</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QSlider" name="cpuSpeed_0">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout">
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="cpuTemp_1">
          <property name="text">
           <string>45°C</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QSlider" name="cpuSpeed_1">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout">
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="cpuTemp_2">
          <property name="text">
           <string>55°C</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QSlider" name="cpuSpeed_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout">
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="cpuTemp_3">
          <property name="text">
           <string>65°C</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QSlider" name="cpuSpeed_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout">
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="cpuTemp_4">
          <property name="text">
           <string>75°C</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QSlider" name="cpuSpeed_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout">
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="cpuTemp_5">
          <property name="text">
           <string>85°C</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QSlider" name="cpuSpeed_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout">
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="cpuTemp_6">
          <property name="text">
           <string>95°C</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QSlider" name="cpuSpeed_6">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout">
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="cpuTemp_7">
          <property name="text">
           <string>105°C</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QSlider" name="cpuSpeed_7">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_15">
        <item alignment="Qt::AlignTop">
         <widget class="QLabel" name="cpuRPM">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gpuGroup">
     <property name="title">
      <string>GPU</string>
     </property>
     <layout class="QHBoxLayout" stretch="0,0,0,0,0,0,0,0,1">
      <item>
       <layout class="QVBoxLayout">
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="gpuTemp_0">
          <property name="text">
           <string>35°C</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QSlider" name="gpuSpeed_0">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout">
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="gpuTemp_1">
          <property name="text">
           <string>45°C</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QSlider" name="gpuSpeed_1">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout">
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="gpuTemp_2">
          <property name="text">
           <string>55°C</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QSlider" name="gpuSpeed_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout">
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="gpuTemp_3">
          <property name="text">
           <string>65°C</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QSlider" name="gpuSpeed_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout">
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="gpuTemp_4">
          <property name="text">
           <string>75°C</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QSlider" name="gpuSpeed_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout">
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="gpuTemp_5">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>85°C</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QSlider" name="gpuSpeed_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout">
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="gpuTemp_6">
          <property name="text">
           <string>95°C</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QSlider" name="gpuSpeed_6">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout">
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="gpuTemp_7">
          <property name="text">
           <string>105°C</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QSlider" name="gpuSpeed_7">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_15">
        <item alignment="Qt::AlignTop">
         <widget class="QLabel" name="gpuRPM">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
